.coveragerc

```diff
@@ -0,0 +1,18 @@
+# this file is based on the examples provided on scikit-learn's .coveragerc
```

Comment:
At least gneiss and wgs-hgt have this file. it will instruct coveralls not to check __main__. Now we have 100% coverage :)